config/routes.rb

```diff
@@ -23,5 +23,6 @@
   get ':username' => 'users/lists#index', as: :user_profile
   get ':username/:list_slug' => 'users/lists#show', as: :user_list
   get ':username/:list_slug/edit' => 'lists#edit', as: :edit_user_list
+  get ':username/:list_slug/destroy' => 'lists#destroy', as: :destroy_user_list
   get ':username/:list_slug/:reference_id' => 'users/references#show', as: :user_reference
```

Comment:
Once we figure out what these routes are, we should definitely use the `resources` shortcuts, but this is fine for now